### DISCUSSION TITLE
All Commits Associated With Different User

### DISCUSSION THREAD

**Discussion by flippind** [community/community#48407] [New to GitHub]
TLDR: All commits are being attributed to the same, incorrect user, but the commit author email is correct throughout and issue persists even on new repos.

I'm pretty sure this is a backend issue only support will be able to solve (ticket #2027924) but, being a free user, I assume traction will be faster with the community.  I built a new repo yesterday and have been spending some time getting GPG signing going.  While checking the commit signature on GitHub, I noticed the commits are being attributed to another user.  I've no idea who this other user is at all.  I've (re)created the repo several times now (including as private and/or with a completely different name), but all commits get attributed to that same user.  I even went so far as to blow up my local repo (it only had one real commit anyway) but the issue persists.

The author email on all the commits is my GitHub noreply email.  The only emails associated with my account are my own.  Checking the commit author as GitHub sees it (via `<commitURL>.patch`) shows my author email.  The **only** resolutions I can find all [reference](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/troubleshooting-commits/why-are-my-commits-linked-to-the-wrong-user) [incorrect](https://github.com/orgs/community/discussions/46003) [author](https://github.com/orgs/community/discussions/24068) [emails](https://github.com/orgs/community/discussions/22149) in some way, shape, or form, which I've summarily ruled out as being the issue.  The [closest I could find](https://github.com/orgs/community/discussions/15753) which didn't only claim bad author email seems to be finger-pointing at GPG key collision but *all* commits, signed or not, are being attributed to this one user.

**Comment by flippind** [comment]
This does seem to be a backend issue.  Support confirmed the user linking process is dropping the first character from the commit author email and escalated the issue.

**Reply by queenofcorgis** [reply]
This is quite interesting - thanks for posting in the community, please keep us updated and good luck!
